Formula/retdec.rb

```diff
@@ -0,0 +1,32 @@
+class Retdec < Formula
+  desc "Retargetable machine-code decompiler based on LLVM"
+  homepage "https://github.com/avast/retdec"
+  url "https://github.com/avast/retdec/archive/refs/tags/v5.0.tar.gz"
+  sha256 "216dc62fd54ff06277497492dbf44bc7a91e39249d8aefdee2e4f10fc903ce85"
+  license all_of: ["MIT", "Zlib"]
+  head "https://github.com/avast/retdec.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
+  depends_on "python@3.11"
+
+  on_macos do
+    depends_on xcode: :build
+    depends_on macos: :mojave
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    assert_match "Running phase: cleanup",
+    shell_output("#{bin}/retdec-decompiler -o #{testpath}/a.c #{test_fixtures("mach/a.out")} 2>/dev/null")
+  end
```

Comment:
That does not look good.
```
$ /opt/homebrew/Cellar/retdec/5.0/bin/retdec-decompiler --version
RetDec version :  HEAD-HASH-NOTFOUND
Commit hash    :  GITDIR-NOTFOUND
Build date     :  2022-12-08T09:46:47Z
```

Reply:
Only way to fix this is by cloning the git repo instead of downloading the release, which I fixed in the new edit. I used the 5.0 release commit so we should be good.